roles/ffh.mesh_wireguard/templates/mesh-vpn.conf.j2

```diff
@@ -0,0 +1,4 @@
+[Match]
+Name=mesh-vpn-{{ '%02d' % domain.id | int }}
```

Comment:
We could simplify this by having only one network file, that matches `Name=mesh-vpn-*`, don't we? Maybe we should prefer this to decrease pollution of /etc/systemd/network, which is already quite a mess.

Reply:
Not sure, I think I need these in order for the network interfaces to come up at all.

Reply:
willst du das auch für wireguard network files?

Reply:
done in fa9572b59549c330e94ea3cfc8e431431fbc5f33